azure-pipelines.vrt-pr.yml

```diff
@@ -1,5 +1,5 @@
 pr:
-  - master
+  - web-components-v3
```

Comment:
This should be resolved to keep what's in `master` branch instead of `web-components-v3`

Reply:
and append the code below https://github.com/microsoft/fluentui/blob/3287714774098eb5f902d9110a214033807d5e34/azure-pipelines.vrt-pr.yml#L18-L95